tests/e2e/k8s/util/kind.go

```diff
@@ -151,6 +154,11 @@ func (c *KindCluster) initializeClients() error {
 		return fmt.Errorf("unable to initialize k8s clientset: %w", err)
 	}
 
+	// Add instance CRD to scheme.
```

Comment:
```suggestion
	// Add clusterlink CRDs to scheme.
```

Also, my personal reference is to rename an import (i.e. `v1alpha1` to `clusterlink`) only when package names collide. But your call.